src/java/fr/paris/lutece/plugins/forms/web/form/column/display/impl/FormColumnDisplayFormResponseDateUpdate.java

```diff
@@ -0,0 +1,100 @@
+ * SUBSTITUTE GOODS OR SERVICES; LOSS OF USE, DATA, OR PROFITS; OR BUSINESS
+ * INTERRUPTION) HOWEVER CAUSED AND ON ANY THEORY OF LIABILITY, WHETHER IN
+ * CONTRACT, STRICT LIABILITY, OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE)
+ * ARISING IN ANY WAY OUT OF THE USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE
+ * POSSIBILITY OF SUCH DAMAGE.
+ *
+ * License 1.0
+ */
+package fr.paris.lutece.plugins.forms.web.form.column.display.impl;
+
+import java.util.Date;
+import java.util.LinkedHashMap;
+import java.util.Locale;
+import java.util.Map;
+
+import fr.paris.lutece.plugins.forms.business.form.column.FormColumnCell;
+import fr.paris.lutece.plugins.forms.business.form.search.FormResponseSearchItem;
+import fr.paris.lutece.portal.service.template.AppTemplateService;
+
+/**
+ * Implementation of the IFormColumnDisplay for the column of the update date of a form response
+ */
+public class FormColumnDisplayFormResponseDateUpdate extends AbstractFormColumnDisplay
+{
+    // Templates
+    private static final String FORM_COLUMN_HEADER_TEMPLATE = "admin/plugins/forms/multiview/column/header/form_column_form_response_date_update_header.html";
+    private static final String FORM_COLUMN_CELL_TEMPLATE = "admin/plugins/forms/multiview/column/cell/form_column_form_response_date_update_cell.html";
+
+    // Marks
+    private static final String MARK_FORM_RESPONSE_DATE_UPDATE_COLUMN_TITLE = "column_title";
+    private static final String MARK_SORT_URL = "sort_url";
+    private static final String MARK_FORM_RESPONSE_DATE_UPDATE = "form_response_date_update";
+    private static final String MARK_COLUMN_SORT_ATTRIBUTE = "column_sort_attribute";
+
+    /**
+     * {@inheritDoc}
+     */
+    @Override
+    public String buildFormColumnHeaderTemplate( String strSortUrl, Locale locale )
+    {
+        Map<String, Object> model = new LinkedHashMap<>( );
+        model.put( MARK_SORT_URL, buildCompleteSortUrl( strSortUrl ) );
+        model.put( MARK_FORM_RESPONSE_DATE_UPDATE_COLUMN_TITLE, getFormColumnTitle( locale ) );
+        model.put( MARK_COLUMN_SORT_ATTRIBUTE, FormResponseSearchItem.FIELD_DATE_UPDATE );
+
+        String strFormResponseDateCreationHeaderTemplate = AppTemplateService.getTemplate( FORM_COLUMN_HEADER_TEMPLATE, locale, model ).getHtml( );
+        setFormColumnHeaderTemplate( strFormResponseDateCreationHeaderTemplate );
+
+        return strFormResponseDateCreationHeaderTemplate;
+    }
+
+    /**
+     * {@inheritDoc}
+     */
+    @Override
+    public String buildFormColumnCellTemplate( FormColumnCell formColumnCell, Locale locale )
+    {
+        Date dateFormResponseDateUpdate = null;
+        if ( formColumnCell != null )
+        {
+            Object objFormResponseDateCreation = formColumnCell.getFormColumnCellValueByName( FormResponseSearchItem.FIELD_DATE_UPDATE );
+            if ( objFormResponseDateCreation != null )
+            {
+                String strTimestamp = String.valueOf( objFormResponseDateCreation );
```

Comment:
The name objFormResponseDateCreation should be replaced by objFormResponseDateUpdate